api/api.py

```diff
@@ -1,10 +1,17 @@
-from flask import Flask 

```

Comment:
When I ran your branch locally, the "What's Nearby" tab doesn't work. Please delete the db implementation of login and use the version of `api.py` in master. Note that what you have right now is missing

```python
from flask import Flask 
app = Flask(__name__) 
```

which is why the tab doesn't work.

Reply:
done